test/integration/org/controls/gcloud.rb

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+scc_notifications_project_id = attribute('scc_notifications_project_id')
+scc_notification_name = attribute('scc_notification_name')
+parent_resource_type = attribute('parent_resource_type')
+parent_resource_id = attribute('parent_resource_id')
+org_id = attribute('org_id')
+domains_to_allow = attribute('domains_to_allow')
+
+boolean_policy_constraints = [
+  'constraints/compute.disableNestedVirtualization',
+  'constraints/compute.disableSerialPortAccess',
+  'constraints/compute.disableGuestAttributesAccess',
+  'constraints/compute.vmExternalIpAccess',
+  'constraints/compute.skipDefaultNetworkCreation',
+  'constraints/compute.restrictXpnProjectLienRemoval',
+  'constraints/sql.restrictPublicIp',
+  'constraints/iam.disableServiceAccountKeyCreation',
+  'constraints/storage.uniformBucketLevelAccess'
+]
+
+control 'gcloud' do
```

Comment:
If possible, we should consider using the native inspec resources for this https://github.com/inspec/inspec-gcp/blob/master/docs/resources/google_organization_policy.md